apps/extension/package.json

```diff
@@ -40,13 +40,16 @@
     "@ledgerhq/hw-transport": "^6.30.0",
     "@ledgerhq/hw-transport-webhid": "^6.28.0",
     "@ledgerhq/hw-transport-webusb": "^6.28.0",
+    "@material-ui/core": "^4.12.4",
```

Comment:
Is that necessary ? It drags an old version babel/runtime with it, which combine are quite huge, and an open door for potentail security issues through babel runtime : https://github.com/advisories/GHSA-67hx-6x53-jw92